packages/react/src/index.ts

```diff
@@ -120,7 +120,7 @@ export {
 export { default as Ref, RefProps } from './components/Ref/Ref'
 export { default as Segment, SegmentProps } from './components/Segment/Segment'
 
-export { default as Status, StatusPropsWithDefaults, StatusProps } from './components/Status/Status'
+export { default as Status, StatusProps } from './components/Status/Status'
```

Comment:
`StatusPropsWithDefaults` was part of public API, so this is a breaking change, too